### PR TITLE
feat(community): adapt open meeting infos

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

This PR adds new open meeting to our tractus-x community section

image

Additional to the content extension, this PR also adds more functionality to the meeting infos

1. possibility to add a session link (like teams, zoom, jitsi)
2. possibility to provide an ics file (this file needs to be created manually)

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: src/components/OpenMeetings/styles.module.css
Comment (SebastianBezold) [inline src/components/OpenMeetings/styles.module.css:1]: Is there a specific reason to move this part of CSS to its own flie? Why not continue with the in-file approach?

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: static/meetings/[C-X] Alignment & Open Planning Trace-X (Follow-up).ics
[the remark this answers is not in the document]
Comment (stephanbcbauer) [reply static/meetings/[C-X] Alignment & Open Planning Trace-X (Follow-up).ics:1]: sounds good. will change it

File: src/components/OpenMeetings/styles.module.css
Comment (stephanbcbauer) [reply src/components/OpenMeetings/styles.module.css:1]: wanted to start to modularize it. But yes, in file is also fine. Will adapt it. thx

**Review by jzbmw — APPROVED** [review]

**Comment by stephanbcbauer** [comment]
@SebastianBezold you offered support, :) I think I need it :) with bringing the style into the file … the list item images doesn't work any more. Now it looks like:

image

I tried differnt thins with this listStyleImage: 'url(\"../../../static/img/product_link_bullet.png\")' but the output in css is always strange.

**Comment by stephanbcbauer** [comment]
> > @SebastianBezold you offered support, :) I think I need it :) with bringing the style into the file … the list item images doesn't work any more. Now it looks like:
> > image
> > I tried differnt thins with this listStyleImage: 'url("../../../static/img/product_link_bullet.png")' but the output in css is always strange.
> 
> sure, how did you want it to look like? Like before, where every line had a white bullet point?
> 
> image

like it was :)

image

In this case it looks like the prodct page :) i think there is only a problem with my property listStyleImage

**Comment by SebastianBezold** [comment]
@stephanbcbauer this should do the trick `listStyleImage: 'url("/img/product_link_bullet.png")'`
I guess the directory resolve + `url(..)` is not handled properly. But every asset in the `static` folder is specially treated and actually does not need to be imported by path

**Comment by stephanbcbauer** [comment]
> @stephanbcbauer this should do the trick `listStyleImage: 'url("/img/product_link_bullet.png")'` I guess the directory resolve + `url(..)` is not handled properly. But every asset in the `static` folder is specially treated and actually does not need to be imported by path

nice. you are right. fixed it. thx

**Review by SebastianBezold — APPROVED** [review]

**Review by mkanal — APPROVED** [review]

**Pull request merged**